### PR TITLE
Docs: made changes in the kpt book as of command reference

### PR DESCRIPTION
**Pull request by NagaHarshita**
Updated the kpt book example commands w.r.t the changes made in the command reference.

### CODE REVIEW

**Review by frankfarzan — APPROVED** [review]

File: site/book/03-packages/02-exploring-a-package.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ In addition, you can use a kpt function such as `search-replace` to run a query
 For example, to search for resources that have a field with path `spec.selector.tier`:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/search-replace:unstable wordpress -- 'by-path=spec.selector.tier'
+$ kpt fn eval wordpress --image gcr.io/kpt-fn/search-replace:unstable -- 'by-path=spec.selector.tier'
```
Comment (phanimarupaka) [inline site/book/03-packages/02-exploring-a-package.md:30]: gcr.io/kpt-fn/search-replace:v0.1

File: site/book/03-packages/03-editing-a-package.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ For example, setting a label on all the resources in the `wordpress` package can
 using the following function:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/set-labels:v0.1 wordpress -- env=dev
+$ kpt fn eval wordpress --image gcr.io/kpt-fn/set-label:v0.1 -- env=dev
```
Comment (phanimarupaka) [inline site/book/03-packages/03-editing-a-package.md:50]: gcr.io/kpt-fn/set-labels:v0.1

File: site/book/04-using-functions/02-imperative-function-execution.md [hunk 3]
```diff
@@ -137,7 +137,7 @@ For instance, the following will set the labels on all resources in the `wordpre
 the `Kptfile`:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/set-labels:v0.1 --include-meta-resources wordpress -- app=wordpress env=prod
+$ kpt fn eval wordpress --image gcr.io/kpt-fn/set-label:v0.1 --include-meta-resources -- app=wordpress env=prod
```
Comment (phanimarupaka) [inline site/book/04-using-functions/02-imperative-function-execution.md:140]: gcr.io/kpt-fn/set-labels:v0.1

**Review by phanimarupaka — CHANGES_REQUESTED** [review]

File: site/book/03-packages/02-exploring-a-package.md (hunk 1 above)
Comment (NagaHarshita) [reply site/book/03-packages/02-exploring-a-package.md]: The following commit has the changes that are suggested.

**Review by phanimarupaka — APPROVED** [review]

**Pull request merged**